app/repositories/submission_repository.rb

```diff
@@ -1,18 +1,26 @@
 class SubmissionRepository
   def rejected
-    Submission.where(rejected: true)
+    Submission.where(rejected: true).order('created_at DESC')
   end
 
   def valid
     Submission.where(rejected: false)
```

Comment:
So I'd probably create some private method like `#not_rejected` which could be reused and have another one - `#valid` order it so it can be ready for a SubmissionPresenter. What do you think?